### ISSUE TITLE
Money is the God of reality, and it is the duty of merchant to make money

### ISSUE THREAD

**Issue by TONGZHENGSHIJIE** [TONGZHENGSHIJIE/-#228]
Money gives light to the world, and money gives warmth to all beings. Money makes people who speaks ill of others have a hard tongue, and money stunns the man who raised the butcher's knife. Money bought a gift for God and knocked on the closed door of God. --"Talmud"

Money is not only the meaning of wealth for Jews. The money is between life and death, the central place of their lives, is the sign of their success.

Whether in ancient times or in modern times, the role of money in society cannot be underestimated. The Jews said, "The rich relatives are close relatives, the poor relatives are distant relatives." The history of the Jews has repeatedly verified this fact. When they don't have money, they are at the bottom of society, people look down on them, and they are insulted and oppressed wherever they go. And when they have money, they can sit as equals at the same table as the nobility, and let people admire and jealous of them. The Jews finally realized that in society, those who have no money are destined to be poor people, and to get dignity and respect, you have to be rich.

After World War II, Jewish soldiers were always discriminated against for no reason at the United Nations Command in Japan, without any dignity. As long as the Jewish soldiers passed by, the white soldiers must scold the "Jewish ghost" with hatred and contempt. Anyone can poke sarcasm at the Jewish soldiers, and the Jewish soldiers are annoyed but helpless.

There was a Jew named Wilson, who was discriminated against by white soldiers and senior officers because of his low rank. Everyone looked down on him, and they often talked about him behind his back. He also tasted all kinds of insults against him. But he has the wisdom of the Jews. At the beginning, he had no money in his pocket. He saved a small amount of money. Then he borrowed the money. It is common for white soldiers to spend a lot of money, and they are always cash-strapped when they can't wait for their salaries. They saw that Wilson had money and could not wait to borrow it from him.

Wilson lent them money and asked them to pay it off within a month with high interest rates, but the soldiers could not handle themselves. After receiving these interest rates, Wilson always always kept saving and lend to the soldiers. For those who have no money to pay back, Wilson lets them mortgage some of their own valuable things and then sell them at a high price. In this way, after a short while, Wilson has lived a prosperous life. He also bought two cars and villas, and he became a "tycoon" in the soldiers. These treatments may not be available even for senior officers. The white soldiers, who often lived in poverty and difficulty, were no longer cocky about Wilson. They were envious of Wilson. Wilson earned his dignity with his wealth.
  
There are many ways for Jews to make money crazily. In the field of blockchain technology, the Jewish saint JT has built a cryptoasset, Jewish businessman AJM, based on the fourth-generation blockchain technology based on the summary of the past blockchain 1.0, 2.0, and 3.0. Its technical advantages are mainly reflected in: Ai enhanced cross-chain technology architecture, distributed secure storage, data non-tamperable, anonymous transactions, lightning network, open third-party ecosystem. Not only that, AJM is also based on the new financial business model, through the integration of payment settlement and UT / ST dual-token issuance, to achieve the digitization or programmization of currency, securities, banking service, and open up the channel of funds into the real economy, promote the revolution of blockchain reform and corporate governance.

Money can not only buy dignity, but also buy a lot of things that you can imagine, all of which are related to money. With money, you have a way of life that everyone admires, with everyone's compliments and envy for you; You also have the right to speak, "people will listen to the words of rich fools, but no one will listen to the proverbs of the poor wise." Today, money is an important measure of success and the value of life, and has even become the only measure in the eyes of some people.

In short, money is the God of reality, Jewish merchant AJM will teach you the most profitable way to make money.

The following is telegram groups of AJM project, welcome to the groups!
https://t.me/ajmofficial_CN
https://t.me/ajmofficial_VN
https://t.me/ajmofficial_RU
https://t.me/ajmofficial_KR
https://t.me/ajmofficial_JP
https://t.me/ajmofficial_EN
